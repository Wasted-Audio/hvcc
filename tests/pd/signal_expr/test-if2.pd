#N canvas 827 239 734 565 12;
#X obj 60 47 loadbang;
#X obj 60 74 bng 25 250 50 0 empty empty empty 17 7 0 10 #191919 #ffffff #ffffff;
#X obj 60 463 print;
#X obj 76 290 sig~;
#X obj 60 431 snapshot~;
#X obj 60 108 t_delays, f 54;
#X msg 131 158 2.3;
#X msg 274 158 3.1;
#X obj 256 290 sig~;
#X obj 76 347 expr~ if($v1 >= 0 \, 1 \, -1);
#X obj 482 347 expr~ if($v1 > 10 \, 10 \, if($v1 < -10 \, -10 \, $v1));
#X msg 417 158 12;
#X obj 482 290 sig~;
#X obj 417 184 t f f f;
#X obj 274 184 t f f f;
#X obj 131 184 t f f f;
#X obj 256 463 print;
#X obj 256 431 snapshot~;
#X obj 482 463 print;
#X obj 482 431 snapshot~;
#X obj 204 393 del 1;
#X obj 60 393 del 2;
#X obj 256 347 expr~ if($v1 > 0 \, $v1 * 10 \, $v1 - 1);
#X connect 0 0 1 0;
#X connect 1 0 5 0;
#X connect 3 0 9 0;
#X connect 4 0 2 0;
#X connect 5 0 21 0;
#X connect 5 0 20 0;
#X connect 5 0 19 0;
#X connect 5 1 6 0;
#X connect 5 2 21 0;
#X connect 5 2 20 0;
#X connect 5 2 19 0;
#X connect 5 3 7 0;
#X connect 5 4 21 0;
#X connect 5 4 20 0;
#X connect 5 4 19 0;
#X connect 5 5 11 0;
#X connect 6 0 15 0;
#X connect 7 0 14 0;
#X connect 8 0 22 0;
#X connect 9 0 4 0;
#X connect 10 0 19 0;
#X connect 11 0 13 0;
#X connect 12 0 10 0;
#X connect 13 0 3 0;
#X connect 13 1 8 0;
#X connect 13 2 12 0;
#X connect 14 0 3 0;
#X connect 14 1 8 0;
#X connect 14 2 12 0;
#X connect 15 0 3 0;
#X connect 15 1 8 0;
#X connect 15 2 12 0;
#X connect 17 0 16 0;
#X connect 19 0 18 0;
#X connect 20 0 17 0;
#X connect 21 0 4 0;
#X connect 22 0 17 0;
