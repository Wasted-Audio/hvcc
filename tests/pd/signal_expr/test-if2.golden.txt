[@ 2.021] print: 10
[@ 3.021] print: 120
[@ 4.021] print: 1
[@ 6.021] print: 3.1
[@ 7.021] print: 31
[@ 8.021] print: 1
[@ 10.021] print: 2.3
[@ 11.021] print: 23
[@ 12.021] print: 1
